site.yml

```diff
@@ -78,6 +78,36 @@
     - role: percona-backup
       tags: ['infra', 'percona']
 
+- name: install mongodb on primary
+  gather_facts: force
+  hosts: mongo_db[0]
+  vars:
+    mongodb_primary: True
+  roles:
+    - role: mongodb
+      tags: ['mongodb', 'mongodb-primary']
+      when: ceilometer.enabled|bool
+
+- name: install mongodb on secondaries
+  gather_facts: force
+  hosts: mongo_db:!mongo_db[0]
+  vars:
+    mongodb_primary: False
+  roles:
+    - role: mongodb
+      tags: ['mongodb', 'mongodb-secondary']
+      when: ceilometer.enabled|bool
```

Comment:
mongodb/tasks/primary.yml (only run on controller1 currently) needs to run and complete before the next two tasks are called in monogdb/tasks/main.yml (specifically the "add mongodb secondary servers to replica set" task and "add mongodb arbiter to replica set").